Collaboration.md

```diff
@@ -20,6 +20,14 @@ Weekly community meetings will feature regular updates on project happenings, in
 
 If you are new to the InstructLab project, the weekly community meeting is a great place to get started!
 
+### [Weekly InstructLab CLI Meeting](#weekly-instructlab-cli-meeting)
+
+We host weekly InstructLab meetings each Thursday at 14:00 UTC. ([time zone converter](https://www.timeanddate.com/worldclock/meetingdetails.html?year=2024&month=5&day=14&hour=14&min=0&sec=0&p1=37&p2=43&p3=101&p4=224&p5=213&p6=771&p7=248&p8=2))
```

Comment:
Propose rewording to "We host weekly InstructLab CLI meetings" as we are now running into the tool has the same name as the project and can cause confusion problem.